traffic_monitor_golang/common/log/log.go

```diff
@@ -95,6 +97,11 @@ func Eventf(t time.Time, format string, v ...interface{}) {
 	Event.Printf("%.3f %s", float64(t.Unix())+(float64(t.Nanosecond())/1e9), fmt.Sprintf(format, v...))
 }
 
+// EventfRaw writes to the event log with no prefix.
+func EventfRaw(format string, v ...interface{}) {
```

Comment:
No, it's using the Event logger for `access.log`. Which seemed to make sense, accesses are events for this app.